src/main/java/ToDo.java

```diff
@@ -3,8 +3,13 @@ public ToDo(String description) {
         super(description);
     }
 
+    @Override
+    public String toSaveFormat() {
+        return "T|" + (isDone ? "1" : "0") + "|" + description;
+    }
+
     @Override
     public String toString() {
-        return "[T]" + super.toString();
+        return "[T]" + (isDone ? "[X] " : "[ ] ") + description;
```

Comment:
Perhaps you could add a method to Task class that returns a string with (isDone ? "[X] " : "[ ] "). Then you can call super.thisMethod for easier coding?